Sources/Sentry/SentryPredicateDescriptor.m

```diff
@@ -74,11 +74,17 @@ - (NSString *)expressionDescription:(NSExpression *)predicate
         }
         break;
     case NSConditionalExpressionType:
-        return [NSString
-            stringWithFormat:@"TERNARY(%@,%@,%@)",
-            [self comparisonPredicateDescription:(NSComparisonPredicate *)predicate.predicate],
-            [self expressionDescription:predicate.trueExpression],
-            [self expressionDescription:predicate.falseExpression]];
+            if (@available(macOS 10.11, *)) {
+                return [NSString
+                        stringWithFormat:@"TERNARY(%@,%@,%@)",
+                        [self comparisonPredicateDescription:(NSComparisonPredicate *)predicate.predicate],
+                        [self expressionDescription:predicate.trueExpression],
+                        [self expressionDescription:predicate.falseExpression]];
+            } else {
+                //this is not supposed to happen, NSConditionalExpressionType was introduced in macOS 10.11
+                //but we need this version check because cocoapod lint check is failing without it.
+                return @"";
+            }
```

Comment:
`m`: Wouldn't this happen on macOS 10.10? Now the default is `@""`, is this acceptable? Can't we provide a better fallback?

Reply:
The enum itself does not exists in macOS 10.10, so this switch block will not be called. But without the check pod check fails